### ISSUE TITLE
[BUG] Actions Nested in Frame Traits and Core Actives/Passives Don't Print to Chat

### ISSUE THREAD

**Issue by msprijatelj** [Eranziel/foundryvtt-lancer#498]
**Describe the bug**
Clicking the "Action" button nested in a Mech Sheet fails to print the Action to chat and instead throws an error in the console.

**To Reproduce**
1. Create a new Mech Sheet.
2. Drag the Black Witch frame from the Compendium onto the sheet.
3. Navigate to the `<MECH//LOADOUT>` tab.
4. Click on the "Reaction" button under the Mag Parry action within the Mag Parry trait.
5. Observe that nothing is printed to chat.
6. Repeat steps 2-5 with Hydra, instead clicking the "Quick" button for each Drone.
7. Repeat steps 2-6 with Mourning Cloak, instead clicking the "Full" button for Blinkspace Jump.

**Expected behavior**
Clicking the "Action" button for an action nested in a frame trait or core passive/active prints the action to chat (and potentially hooks into the action tracker functionality).

**Screenshots**
Screenshot is from the 1.4.0-beta.1 prerelease, but the bug is also present on Lancer 1.3.1.
![image](https://user-images.githubusercontent.com/44978587/210700604-8000d049-9b65-4d86-99c0-c66cf63dbbc4.png)

**Platform (please complete the following information):**
 - OS: Windows 10
 - Browser: Foundry App
 - Foundry version v9.280 and v10.291
 - Lancer system version v1.3.1 and v1.4.0-beta.1

**Additional context**
If Zheng from the Long Rim is installed, using Zheng to test will save some time since it has a Trait with a Free action and a Core Passive with a Free Action.

**Discord contact info**
Valkyrion#6196

**Comment by Davis-K-Johnson** [comment]
I just tested this and I'm not sure if it's fixed as the reporter intended or not. In the current version mag parry doesn't even have a reaction sub-popout at all. Hydra has all the drones listed but no buttons to activate them. Not sure if that's intended or not. The Use full button on the Mourning Cloak is working as intended.

![Image](https://github.com/Eranziel/foundryvtt-lancer/assets/89803504/70139926-9662-4841-9976-519e27bf1283)



![Image](https://github.com/Eranziel/foundryvtt-lancer/assets/89803504/1d79ce2c-b2e6-4bf5-80f2-d0d7532d85f9)



![Image](https://github.com/Eranziel/foundryvtt-lancer/assets/89803504/63976529-f280-4e25-9b0a-b165d7a01061)



![Image](https://github.com/Eranziel/foundryvtt-lancer/assets/89803504/cd448cb0-a394-416d-beb9-03090f71d4d2)

I also tested it with the mimic mesh system from Gorgon. Instead of submenus in the popout it just has separate buttons, which works well and is intuitive.
